### PR TITLE
keras.layers.Bidirectional constructor ignores trainable argument, causing models with trainable=False layers to incorrectly be cloned 

### LINKED ISSUE #17466: keras.layers.Bidirectional constructor ignores trainable argument, causing models with trainable=False layers to incorrectly be cloned
**System information**.
- Have I written custom code (as opposed to using a stock example script provided in Keras): no
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): CentOS
- TensorFlow installed from (source or binary): PYPI
- TensorFlow version (use command below): 2.11.0
- Python version: 3.7

**Describe the problem**.

`keras.layers.Bidirectional` ignore the `trainable` argument, therefore cloning a model with Bidirectional layers, where the layer has been set to `trainable=False` will be cloned with `trainable` set to `True`.

**Describe the current behavior**.
The `trainable` property is set always to `True`.

**Describe the expected behavior**.
The `trainable` property should be set correctly.

**[Contributing](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md)**.

- Do you want to contribute a PR? (yes/no): yes
- If yes, please read [this page](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md) for instructions
- Briefly describe your candidate solution(if contributing):

Change `trainable = True` to `trainable = kwargs.get('trainable', True)` in https://github.com/keras-team/keras/blob/master/keras/layers/rnn/bidirectional.py#L178

**Standalone code to reproduce the issue**.

```python
import tensorflow as tf

inp = tf.keras.Input([None, 3])
rnn = tf.keras.layers.SimpleRNN(units=3)
bid = tf.keras.layers.Bidirectional(rnn)
model = tf.keras.Model(inp, bid(inp))

model.summary()

clone_trainable = tf.keras.models.clone_model(model)
assert clone_trainable.get_config() == model.get_config()

bid.trainable = False
clone_untrainable = tf.keras.models.clone_model(model)
assert clone_untrainable.get_config() == model.get_config()
```

### PR DESCRIPTION
**Pull request by mergian**
This PR fixes #17466.

### CODE REVIEW

File: keras/layers/rnn/bidirectional_test.py [hunk 1]
```diff
@@ -1020,6 +1020,20 @@ def test_full_input_spec(self):
...
         self.assertNotAllClose(output1, output2)
 
+    def test_trainable_parameter_argument(self):
+        inp = keras.layers.Input([None, 3])
+        rnn = keras.layers.SimpleRNN(units=3)
+        bid = keras.layers.Bidirectional(rnn)
+        model = keras.Model(inp, bid(inp))
+
+        clone_trainable = keras.models.clone_model(model)
+        assert clone_trainable.get_config() == model.get_config()
```
Comment (qlzh727) [inline keras/layers/rnn/bidirectional_test.py:1030]: Please use `self.assertEqual()` instead.

File: keras/layers/rnn/bidirectional_test.py [hunk 2]
```diff
@@ -1020,6 +1020,20 @@ def test_full_input_spec(self):
...
+
+        bid.trainable = False
+
+        clone_untrainable = keras.models.clone_model(model)
+        assert clone_untrainable.get_config() == model.get_config()
```
Comment (qlzh727) [inline keras/layers/rnn/bidirectional_test.py:1035]: Same here.

File: keras/layers/rnn/bidirectional_test.py [hunk 3]
```diff
@@ -1020,6 +1020,20 @@ def test_full_input_spec(self):
...
+
```
Comment (qlzh727) [inline keras/layers/rnn/bidirectional_test.py:1036]: Can you also add a test to create a `keras.layers.Bidirectional` with a trainable = False in the __init__ arg?

**Review by qlzh727 — CHANGES_REQUESTED** [review]
Thanks for the PR.

**Comment by mergian** [comment]
I changed `self._trainable` to `= kwargs.get('trainable', layer.trainable)` as also other attributes get fetched from the layer. I also changed the test cases to test for these behavior.

**Review by qlzh727 — APPROVED** [review]

**Pull request merged**